src/webgpu/api/operation/queue/writeBuffer.spec.ts

```diff
@@ -8,3 +8,89 @@ import { makeTestGroup } from '../../../../common/framework/test_group.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export const g = makeTestGroup(GPUTest);
+
+g.test('writeBuffer,Ranges')
+  .params([
+    {},
+  ])
+  .fn(async t => {
+    const queue = t.device.defaultQueue;
+
+    function runTest(arrayType : any, testBuffer : boolean) {
```

Comment:
You can add an `export` on this type:
https://github.com/gpuweb/cts/blob/main/src/webgpu/gpu_test.ts#L25
and use it here.